contrib/sarplus/python/pysarplus/SARPlus.py

```diff
@@ -25,7 +21,7 @@
 
 
 class SARPlus:
-    """SAR implementation for PySpark"""
+    """SAR implementation for PySpark."""
 
     def __init__(
```

Comment:
I think it will be useful to have docstrings in the init function. In the CPU SAR there should be information that can be reused

Reply:
Added